cmd/plume/containerlinux.go

```diff
@@ -90,142 +90,89 @@ var (
-				azureEnvironments,
-				"publish",
-				"Flatcar Edge",
-				"_pro",
-				"The Edge channel closely tracks current development work and is released frequently. The newest versions of the Linux kernel, systemd, and other components will be available for testing.",
-			),
-			AWS: newAWSSpec(),
+			GCE:            newGceSpec("edge", edge_desc),
+			Azure:          newAzureSpec(azureEnvironments, "publish", "Flatcar Edge", "", edge_desc),
+			AzurePremium:   newAzureSpec(azureEnvironments, "publish", "Flatcar Edge", "_pro", edge_desc),
+			AWS:            newAWSSpec(),
 		},
 		"lts": channelSpec{
 			BaseURL:        "gs://flatcar-jenkins-private/lts/boards",
 			BasePrivateURL: "gs://flatcar-jenkins-private/lts/boards",
 			Boards:         []string{"amd64-usr"},
 			Destinations:   []storageSpec{},
-			GCE:            gceSpec{},
-			Azure: newAzureSpec(
-				azureEnvironments,
-				"publish",
-				"Flatcar LTS",
-				"",
-				"The LTS channel should be used by production clusters. Versions of Flatcar Container Linux are battle-tested within the Stable channel before being promoted.",
-			),
-			AzurePremium: newAzureSpec(
-				azureEnvironments,
-				"publish",
-				"Flatcar LTS",
-				"_pro",
-				"The LTS channel should be used by production clusters. Versions of Flatcar Container Linux are battle-tested within the Stable channel before being promoted.",
-			),
-			AWS: newAWSSpec(),
+			GCE:            newGceSpec("lts", lts_desc),
+			Azure:          newAzureSpec(azureEnvironments, "publish", "Flatcar LTS", "", lts_desc),
+			AzurePremium:   newAzureSpec(azureEnvironments, "publish", "Flatcar LTS", "_pro", lts_desc),
+			AWS:            newAWSSpec(),
 		},
 		"developer": channelSpec{
 			BaseURL:        "gs://flatcar-jenkins/developer/developer/boards",
 			BasePrivateURL: "gs://flatcar-jenkins-private/developer/developer/boards",
 			Boards:         []string{"amd64-usr", "arm64-usr"},
 			Destinations:   []storageSpec{},
 			GCE:            gceSpec{},
-			Azure: newAzureSpec(
-				azureEnvironments,
-				"developer",
-				"Flatcar Developer Channel",
-				"",
-				"The Developer Channel is used for internal test builds.",
-			),
-			AzurePremium: newAzureSpec(
-				azureEnvironments,
-				"developer",
-				"Flatcar Developer Channel",
-				"_pro",
-				"The Developer Channel is used for internal test builds.",
-			),
-			AWS: newAWSSpec(),
+			Azure:          newAzureSpec(azureEnvironments, "developer", "Flatcar Developer Channel", "", dev_desc),
+			AzurePremium:   newAzureSpec(azureEnvironments, "developer", "Flatcar Developer Channel", "_pro", dev_desc),
+			AWS:            newAWSSpec(),
 		},
 	}
 )
 
+func newGceSpec(channel, description string) gceSpec {
+	return gceSpec{
+		Project:     "kinvolk-public",
+		Family:      fmt.Sprintf("flatcar-%s", channel),
+		Description: description,
+		Licenses:    []string{"flatcar-container-linux"},
+		Image:       "flatcar_production_gce.tar.gz",
+		Publish:     "",
+		Limit:       50,
```

Comment:
Is the file still needed after publishing? Otherwise we could reduce this even more.

Reply:
I'm not sure I understand your question. The Limit variable limits how many images are stored in the account. I wasn't sure which number to pick and it can definitely be made smaller. The files are needed if we want to let people install previous versions.

If you meant whether the file is needed after *installing*, I'm pretty sure it's not. The OS is copied onto the hard drive of the machine, and so removing the image shouldn't affect already running instances.

Reply:
I've reduced the limit to 10 images.